contractrequester/contractrequester.go

```diff
@@ -145,6 +146,8 @@ func (cr *ContractRequester) CallMethod(ctx context.Context, base core.Message,
 		return res, nil
 	}
 
+	ctx, cancel := context.WithTimeout(ctx, configuration.NewAPIRunner().Timeout)
```

Comment:
we have same situation in callconstructor, add it there too, pls.

Reply:
added, thinling about test suites